frontend/src/app/commonComponents/QueryWrapper.tsx

```diff
@@ -56,7 +54,7 @@ export function QueryWrapper<ComponentProps>({
   };
   const props = ({
     ...componentProps,
-    trackAction,
+    trackAction: appInsights?.trackEvent({ name: "User Action" }),
```

Comment:
As a matter of fact I can't find any evidence that `trackAction` is ever called downstream! I imagine this prop could be removed without incident, but it might be safer to supply a no-op function here in the case that we don't have an App Insights client available.

Reply:
If it's unused then this change is fine by me :)